### DISCUSSION TITLE
Hosting Zone Issue

### DISCUSSION THREAD

**Discussion by krn010** [learndevopsonline/learndevopsonline#1901] [Q&A]
I bought a domain and mapped the IP's of frontend backend and mysql to the created hosting zones. But when i replaced the names with the ip and tried starting nginx.  It failed to start whereas with the private ip reverse proxy works. What could be the cause?

**Comment by chaithanya1812** [comment]
Hi @krn010 , you bought the domain today?
Check you domin in dns checker org website.
Name your domain in chat?

**Reply by krn010** [reply]
No domain was bought 5 dayws back. Its already registered in dns checker site. krndevops.online

**Reply by chaithanya1812** [reply]
### Issue :-  update the Name-servers in hostinger.

**Reply by krn010** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/30590696/d6951561-3ff1-416a-943e-9955cfe03376)

**Reply by krn010** [reply]
Can you share the video link, cant find it anymore

**Reply by krn010** [reply]
Video link of hostinger

**Reply by chaithanya1812** [reply]
Which batch you from?

**Reply by krn010** [reply]
77

**Reply by chaithanya1812** [reply]
Watch session -11 video

**Reply by chaithanya1812** [reply]
If not resolved , you join lab session..?
